src/pages/Beta/Pool/DetailModal/StatDetail/index.tsx

```diff
@@ -41,14 +41,18 @@ export default function StatDetail({ title, totalAmount, pair, pgl, currency0, c
       </Text>
 
       <StateContainer>
-        <Stat
```

Comment:
why are we not using Stat component ?

Reply:
so that it doesn't change other places where this component is used